### DISCUSSION TITLE
How to use web-tree-sitter in a website?

### DISCUSSION THREAD

**Discussion by ghost** [tree-sitter/tree-sitter#3034] [Q&A] [answered]
I am considering build my online real time transcompiler on a website. I want to know if it is possible to implement my website with node front end like Angular.JS or React.JS along with web-tree-sitter.

**Comment by ahelwer** [comment, marked as answer]
Here's how the playground consumes a WASM tree-sitter grammar: https://github.com/tree-sitter/tree-sitter/blob/master/cli/src/playground.html

Here's my customized version for my own grammar: https://github.com/tlaplus-community/tlaplus-community.github.io/tree/main/tree-sitter-tlaplus

Since this isn't a bug report it is probably best suited for a [discussion](https://github.com/tree-sitter/tree-sitter/discussions).
